tests/pkcli/ci1_test.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""test ci
+
+:copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import pkg_resources
+from pykern.pkdebug import pkdp
+import re
+import pytest
+
+
+def test_check_prints_dot_prefix():
+    from pykern.pkcli import ci, test
+    from pykern import pkunit
+
+    ci._EXCLUDE_FILES = re.compile(
+        re.sub(
+            f"/{test.SUITE_D}/.*(?:{pkunit.DATA_DIR_SUFFIX}|{pkunit.WORK_DIR_SUFFIX})/",
```

Comment:
Don't use re.sub when there are specials without escaping the pattern, use string.replace. Also the search pattern can be greatly simplified and probably should be asserted, so:

```py
def _unexclude_work_dir():
    p = ci._EXCLUDE_FILES.pattern.replace(f"|{pkunit.WORK_DIR_SUFFIX}", "")
    assert p != ci._EXCLUDE_FILES.pattern
    ci._EXCLUDE_FILES = re.compile(p, flags=ci._EXCLUDE_FILES.flags)
```